src/import/load-costume.js

```diff
@@ -212,8 +212,17 @@ const loadBitmap_ = function (costume, runtime, _rotationCenter) {
             return fetched;
         })
         .then(({canvas, mergeCanvas, rotationCenter}) => {
-            // createBitmapSkin does the right thing if costume.bitmapResolution or rotationCenter are undefined...
-            costume.skinId = runtime.renderer.createBitmapSkin(canvas, costume.bitmapResolution, rotationCenter);
+            // createBitmapSkin does the right thing if costume.bitmapResolution is undefined...
+            // TODO: is rotationCenter ever undefined?
+            let center;
+            if (rotationCenter) {
+                center = [
+                    rotationCenter[0] / 2,
+                    rotationCenter[1] / 2
```

Comment:
Note that it's okay to always divide by 2 here-- [`_fetchBitmapCanvas` always scales up the rotation center](https://github.com/LLK/scratch-vm/blob/8faad674396c906596e6a678c0c11fa4aa9f1333/src/import/load-costume.js#L154).

Reply:
Could you add this as a comment in the code?